include/ibm/flight_recorder.hpp

```diff
@@ -0,0 +1,116 @@
+#pragma once
+
+#include <fstream>
+#include <iomanip>
+#include <iostream>
+#include <vector>
+
+#define FLIGHT_RECORDER_MAX_ENTRIES 300
```

Comment:
Shouldn't this `FLIGHT_RECORDER_MAX_ENTRIES (or flightRecorderMaxEntries)` be larger than max connection count at least to hold any potential pending connections?

https://github.com/ibm-openbmc/bmcweb/blob/ac4fd01279f5f8873cca21c3e16c16da5d7d2cd6/http/http_connection.hpp#L294

Perhaps should we introduce maxConnectionCount constant and use it to define flightRecorderMaxEntries?

```
http_connection.hpp:
constexpr int maxConnectionCount = 200;

L294:
        if (connectionCount >= maxConnectionCount)
```

And here

```
constexpr int flightRecorderMaxEntries = (maxConnectionCount + 100);   //maxcount with some extras
```

Reply:
maxConnectionCount is not all related for redfish evening.